### PR TITLE
IKASAN-1756 sorting out read only role

### PR DESCRIPTION
**Pull request by mick-stewart73**

### CODE REVIEW

File: ikasaneip/visualisation/dashboard/pom.xml [hunk 1]
```diff
@@ -87,6 +87,11 @@
             <artifactId>file-download-wrapper</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>mysql</groupId>
```
Comment (andrzej-majewski) [inline ikasaneip/visualisation/dashboard/pom.xml]: Can we take this one out

File: ikasaneip/visualisation/dashboard/src/main/resources/application.properties [hunk 2]
```diff
@@ -21,12 +21,24 @@ solr.password=1ka5an
...
+#datasource.xadriver-class-name=org.h2.jdbcx.JdbcDataSource
+#datasource.url=jdbc:h2:tcp://localhost:${h2.db.port}/./${module.name}-db/esb;IFEXISTS=FALSE
+#datasource.dialect=org.hibernate.dialect.H2Dialect
+#datasource.show-sql=false
+#datasource.hbm2ddl.auto=none
+#datasource.validationQuery=select 1
+#datasource.min.pool.size=5
+#datasource.max.pool.size=20
+
+datasource.username=root
```
Comment (andrzej-majewski) [inline ikasaneip/visualisation/dashboard/src/main/resources/application.properties]: revert

File: ikasaneip/visualisation/dashboard/pom.xml (hunk 1 above)
Comment (mick-stewart73) [reply ikasaneip/visualisation/dashboard/pom.xml]: done

File: ikasaneip/visualisation/dashboard/src/main/resources/application.properties (hunk 2 above)
Comment (mick-stewart73) [reply ikasaneip/visualisation/dashboard/src/main/resources/application.properties]: done

**Review by andrzej-majewski — APPROVED** [review]

**Pull request merged**